tests/torch/test_hybrid_converter.py

```diff
@@ -55,7 +55,9 @@ def run_hybrid_model_test(
     accuracy = is_in.float().mean()
 
     # Make sure accuracy is above a certain threshold
-    assert accuracy >= expected_accuracy, "Expected accuracy GPT2 hybrid not matched."
+    # We need to add some tolerance otherwise the test is flaky
+    tolerance = 0.005
+    assert accuracy >= expected_accuracy - tolerance, "Expected accuracy GPT2 hybrid not matched."
```

Comment:
Would a single PR with 2 commits be okay with you?

I'll create an issue and add the link here

Reply:
I mean, it feels weird to tackle the hybrid flaky in a PR called "remove fashionmist" so yes I guess it would be better to have a small independent PR (better tracking as well) , but not a hug huge deal anyway